### PR TITLE
Abdul/iot enterprise default list

### PR DESCRIPTION
**Pull request by v-abdrahman**
Created a folder for IoTEnterpriseDefault List & added all the project files for windows 10 IoT Enterprise.

Following changes are done:

1) file modified : SmartDisplay.Contracts.csproj 
    TargetPlatformVersion 0.0.17763.0 To TargetPlatformVersion 10.0.19041.0

2) file modified : SmartDisplay.Features.csproj 
    TargetPlatformVersion 10.0.17763.0 To TargetPlatformVersion 10.0.19041.0

3) file modified : SmartDisplay.Utils/Keys.cs
    IoTCoreDefaultAppUnderTest to IoTEnterpriseDefaultAppUnderTest

4) file modified : SmartDisplay.Utils.csproj
    TargetPlatformVersion 10.0.17763.0 To TargetPlatformVersion 10.0.19041.0

5) file modified : SmartDisplay/Package.appxmanifest
    IoTCoreDefaultAppUnderTest to IoTEnterpriseDefaultAppUnderTest
   
6) File containing IoT Core has been changed to IoT Enterprise

7) String folders - IoT Core is changed to IoT Enterprise

8) file modified : README.md
    Adding windows 10 IoT Enterprise in products.
    Updated the IoT Enterprise Startup image
    Updated the Windows Platform from IoT Core to IoT Enterprise

9) Nuget Packages installed
     Newtonsoft.Json - Version 11.0.2
     Microsoft.NETCore.UniversalWindowPlatform - Version 6.2.9
     Microsoft.Composition - Version 1.0.31

### CODE REVIEW

**Review by rsameser — APPROVED** [review]

File: IoTEnterpriseDefault/CS/SmartDisplay/Utils/UI/ViewLifetimeControl.cs [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+    /// keeps track of when the secondary view thinks it's in usem and when the main view is interacting with the secondary view (about to show
+    /// it to the user, etc.) When the reference count drops to zero, the secondary view is closed.
+    /// </summary>
+    public sealed class ViewLifetimeControl : INotifyPropertyChanged
+    {
+        // Dispatcher for this view. Kept here for sending messages between this view and the main view.
+        public CoreDispatcher Dispatcher { get; private set; }
+
+        // Each view has a unique Id, found using the ApplicationView.Id property or
+        // ApplicationView.GetApplicationViewIdForEnterpriseWindow method. This id is used in all of the ApplicationViewSwitcher
```
Comment (neilsh-msft) [inline IoTEnterpriseDefault/CS/SmartDisplay/Utils/UI/ViewLifetimeControl.cs]: Change GetApplicationViewIdForEnterpriseWindow back to GetApplicationViewIdForCoreWindow.  This is an actual API method name not a reference to IoT Core.

File: IoTEnterpriseDefault/CS/SmartDisplay.Utils/Authentication/WamProvider.cs [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+        private void ClearTokens()
+        {
+            _tokens.Clear();
+            CachedProperties.Clear();
+        }
+
+        private async Task<WebTokenRequestResult> RequestTokenWithTimeout(WebTokenRequest request)
+        {
+            // The WebTokenRequest will time out if the user does not complete in time.
+            // This is because there is currently no 'close' button on IoT Enterprise, so it
```
Comment (neilsh-msft) [inline IoTEnterpriseDefault/CS/SmartDisplay.Utils/Authentication/WamProvider.cs]: There is a 'close' button on IoT Enterprise.  Please change comment back to IoT Core which this comment is applicable to.

**Review by neilsh-msft — CHANGES_REQUESTED** [review]
I tried building this project using Visual Studio 2019 and ran into a number of build errors.  What version of VS did you use, and what .net framework do you have installed?

**Comment by v-abdrahman** [comment]
I have installed VS2019 and Microsoft.NETCore.UniversalWindowPlatform - Version 6.2.9

File: IoTEnterpriseDefault/CS/SmartDisplay/Utils/UI/ViewLifetimeControl.cs (hunk 1 above)
Comment (v-abdrahman) [reply IoTEnterpriseDefault/CS/SmartDisplay/Utils/UI/ViewLifetimeControl.cs]: Made the changes

File: IoTEnterpriseDefault/CS/SmartDisplay.Utils/Authentication/WamProvider.cs (hunk 2 above)
Comment (v-abdrahman) [reply IoTEnterpriseDefault/CS/SmartDisplay.Utils/Authentication/WamProvider.cs]: Made the changes

**Pull request merged**